clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -10283,6 +10283,11 @@ void OffloadWrapper::ConstructJob(Compilation &C, const JobAction &JA,
     CmdArgs.push_back(
         TCArgs.MakeArgString(Twine("-target=") + Triple.getTriple()));
 
+    if (Inputs[0].getType() == types::TY_Tempfiletable ||
+        Inputs[0].getType() == types::TY_Tempfilelist)
+      // wrapper actual input files are passed via the batch job file table:
```

Comment:
`wrapper actual input files...`
This sentence seems odd to me. Can you clarify?

Reply:
yeah, this doesn't read very well.  It should just say 'Input files are passed via the batch job file table'